### PR TITLE
idm_23.4.1_release

### PR DESCRIPTION
**Pull request by manjunathdhegde-2910**

### CODE REVIEW

File: OracleIdentityGovernance/dockerfiles/12.2.1.4.0/container-scripts/set_feurl.py
Comment (Djelibeybi) [inline OracleIdentityGovernance/dockerfiles/12.2.1.4.0/container-scripts/set_feurl.py:1]: Oracle Linux doesn't support unversioned Python, so this is unlikely to work. If you don't invoke this scrpt as an executable and only call it via the `python3` interpreter, you can/should remove this line. If you do want to call it like an executable, it needs to refer to `python3`

Comment (manjunathdhegde-2910) [reply OracleIdentityGovernance/dockerfiles/12.2.1.4.0/container-scripts/set_feurl.py]: Have fixed both linter issue comment and python comment.

**Review by Djelibeybi — APPROVED** [review]

**Comment by Djelibeybi** [comment]
@rishiagarwal-oracle you still need to reapprove this PR since changes have been made after your previous approval.

**Review by rishiagarwal-oracle — APPROVED** [review]

**Pull request merged**
